### PR TITLE
6905 add line item model with corresponding spec

### PR DESCRIPTION
**Pull request by arisah-ijeoma**
In the transaction model, I noticed there were two line items; one for credit and the other for debit. That being said, I'm not sure what the association is from transactions and accounts (i.e. has_many or has_one)
@hooverlunch

### CODE REVIEW

**Comment by linesarefuzzy** [comment]
@arisah-ijeoma `has_many` for both.

**Comment by arisah-ijeoma** [comment]
Turned out it was a db issue causing the bug I had initially

**Comment by emitche** [comment]
I've spoken to @hooverlunch  and he plans to review this by 1 PM. This will be the last issue I QA for the release.

File: app/models/loan_question.rb [hunk 1]
```diff
@@ -9,7 +9,6 @@
 #  internal_name         :string
 #  loan_question_set_id  :integer
 #  migration_position    :integer
-#  number                :integer
```
Comment (hooverlunch) [inline app/models/loan_question.rb]: Not sure why this is getting removed here. Thoughts?

File: db/schema.rb [hunk 2]
```diff
@@ -409,20 +425,20 @@
...
-    t.boolean  "notify_on_new_logs", default: true
+    t.boolean  "notify_on_new_logs",     default: true
     t.integer  "profile_id"
     t.datetime "remember_created_at"
     t.datetime "reset_password_sent_at"
     t.string   "reset_password_token"
-    t.integer  "sign_in_count", default: 0, null: false
-    t.datetime "updated_at", null: false
+    t.integer  "sign_in_count",          default: 0,    null: false
+    t.datetime "updated_at",                            null: false
```
Comment (hooverlunch) [inline db/schema.rb]: I'm not sure how all these spaces got added in. We are using the fix-db-schema-conflicts gem which should prevent this. Can you try running your db:schema:dump again? Definitely don't want all these schema changes in the PR in any case please. Just what you're actually adding.

File: db/schema.rb [hunk 3]
```diff
@@ -436,6 +452,8 @@
 
   add_index "users_roles", ["user_id", "role_id"], name: "index_users_roles_on_user_id_and_role_id", unique: true, using: :btree
 
+  add_foreign_key "accounting_line_items", "accounting_accounts"
+  add_foreign_key "accounting_line_items", "accounting_transactions"
```
Comment (hooverlunch) [inline db/schema.rb]: These look great.

File: app/models/loan_question.rb (hunk 1 above)
Comment (arisah-ijeoma) [inline app/models/loan_question.rb]: I'll run `db:schema:dump`. Perhaps, that's fix this otherwise of course there is another issue

**Pull request merged**